direct_message/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.home, name='Direct-message-home'),
```

Comment:
Although path names aren't [restricted to valid Python names](https://docs.djangoproject.com/en/4.0/topics/http/urls/#naming-url-patterns-1), I still think it should be lowercase